packages/connect-form/src/types/connect-form-tab.ts

```diff
@@ -0,0 +1,6 @@
+export type ConnectFormTab =
```

Comment:
Rather than overloading the role of tab name/label we could use a separate field, like tab id that is only used to match errors with tabs, similar to what we did for field ids.

Considering that we don't have anything in `/types/` yet. I'd suggest doing something like this in  `validation.ts` to keep it consistent with the rest and avoid introducing new patterns: 

```
type FieldTab = 'general' | 'authentication' | 'tls' | 'tunnels' | 'advanced';

export type ConnectionFormError = {
  fieldName?: FieldName;
  fieldIndex?: number;
  fieldTab?: FieldTab;
  message: string;
};
```

then if you add a function like this: 

```
export getErrorsByTab(errors, tabId: FieldTab) {...}
```

you don't even need to export the `FieldTab` type.

Reply:
👍 updated